Kitodo/src/main/resources/messages/messages_de.properties

```diff
@@ -541,7 +541,9 @@ metsRightsOwnerMail=METS Rechteinhaber Kontakt
 metsRightsOwnerSite=METS Rechteinhaber URL
 metsfilegroups=Unterordner und METS Dateigruppen
 migrateMetadata=Metadaten konvertieren
-migrateProjects=Migrieren
+migrateMetadataHint=Die Konvertierung wurde im Taskmanager gestartet.
+migrateMetadataOfSelectedProjects=Metadaten dieser Projekte konvertieren
+migrateProjects=Nach Vorg\u00E4ngen suchen
```

Comment:
This change also changes the name of the tab
![migrationTab](https://user-images.githubusercontent.com/22615787/68285292-4a8fb500-007f-11ea-86cf-16f85c4667cc.PNG)
I think "Migration" would fit better here